test/service/test_async_manifest_service.py

```diff
@@ -145,7 +145,7 @@ def test_manifest_endpoint_start_execution(self, mock_uuid, step_function_helper
                     mock_uuid.return_value = execution_name
                     step_function_helper.describe_execution.return_value = {'status': 'RUNNING'}
                     format_ = 'compact'
-                    filters = {'file': {'organ': {'is': ['lymph node']}}}
+                    filters = {'organ': {'is': ['lymph node']}}
```

Comment:
Was this a bug? Why was this test passing before?

Reply:
Appears to be a bad test, or at least a test that doesn't check the validity of the filters. This test is only verifying the response after the first request to the manifest service, checking for a 301 status and for the 'Retry-After' and 'Location' fields.
I'm not sure why the filter's value used was chosen, the test passes with `filters = {}` or even `filters = {'foo': 'bar'}`

Reply:
Since this uses mocks, the filter doesn't matter, right? This used to be a correct filter, for historic reasons. This predates my time, potentially from before the indexer denormalized associated entities into inner ones.

Reply:
Yes, normally a request to the manifest endpoint is handled by `handle_manifest_generation_request()` which takes the `filters` param and uses it in a call to `ManifestService.get_cached_manifest`. This test function mock's `get_cached_manifest()` to simply return None regardless of what is passed in.